### DISCUSSION TITLE
It is not possible to cancel the download

### DISCUSSION THREAD

**Discussion by trup** [aclap-dev/video-downloadhelper#1068] [Help and Q&A] [answered]
### Your question

![изображение](https://github.com/user-attachments/assets/3ff99e28-8925-485c-a86a-77b6b4bb87b7)

This video has not started downloading for over 1 hour and there is no way to cancel it except restarting the browser.
Need a forced delete button.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: ru
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\AnnA\\Videos",
    "concurrent_downloads_max": 5,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "show_context_menu": false,
    "yt_warning": false,
    "use_wide_ui": false,
    "use_sidebar": false,
    "open_count_store": 28,
    "successfull_dl": 61
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There is a way to cancel a zombie download like what you're seeing.  It's in the `More settings` section of VDH settings.  Scroll it all the way to the bottom.  The `Reload addon` button is what you want.  But be careful about using it.  When you click that button, any inflight downloads that are not stalled will be abruptly terminated leaving you with an unplayable file.  You would have to redo those downloads from the beginning.  So I would wait until all such downloads were done before clicking that button.

A better question might be why your download was stuck for an hour.  Did you lose network connectivity?  Did the server go down?  Something else?

**Reply by trup** [reply]
thanks for your help.
internet works stable, and (this) youtube too.
i don't know the reason for the problem.

**Reply by paulrouget** [reply, marked as answer]
It's possible that, indeed, a download never starts. We should have a way to kill those, but we don't at the moment.

Like @GrampaWildWilly said, you can reload the addon, like so (in the "More Settings" menu):

<img width="326" alt="Screenshot 2024-07-31 at 12 28 38" src="https://github.com/user-attachments/assets/ce573489-216b-4cde-b4e1-66f8c8e69cdc">
